test/test_autotuner_utility.cc

```diff
@@ -54,7 +54,9 @@ std::vector<CudaMappingOptions> restoreCandidates(
   });
 
   return tc::autotune::restoreCandidates(
-      tc, inputsPair.first, outputsPair.first);
+      tc::autotune::canonicalTC(lang::Parser(tc).parseFunction()),
```

Comment:
Should we add an  `std::string` overload to `tc::autotune::canonicalTC` to handle this case?